src/GafferSceneUI/SceneView.cpp

```diff
@@ -843,6 +859,28 @@ class CameraOverlay : public GafferUI::Gadget
 				glColor4f( 0, 0.25, 0, 1.0f );
 				style->renderRectangle( m_resolutionGate );
 
+				if( m_overscan[0] != 0.0f || m_overscan[1] != 0.0f || m_overscan[2] != 0.0f || m_overscan[3] != 0.0f )
+				{
+					glLineWidth( 0.5f );
+					glLineStipple( 1, 0x3333 );
+					glEnable( GL_LINE_STIPPLE );
```

Comment:
It's possible that my ageing eyesight is to blame, or my monitor, but I can barely see the overlay. It only saw it for the first time after I had instrumented the code to see if it was actually getting called! Screenshot below :

![image](https://user-images.githubusercontent.com/1133871/74176487-af831a00-4c2f-11ea-8040-cd79b007a59b.png)

I can see the thick green resolution gate and the thick red aperture just fine, but the overscan seems a little shy.

Reply:
Totally - that's impossible to see! Seems to be some variation across machines (compared to the screen shot at the top).See if this is better in https://github.com/GafferHQ/gaffer/pull/3584/commits/3334962d195251fc1f9de8a6db050632cabb06e0. It now the same as the resolution gate, except stippled.

![image](https://user-images.githubusercontent.com/896779/74224220-53140f00-4cb0-11ea-8b93-e49e420372a6.png)

Reply:
Hallelujah! I can see again!